docs/source/user_guide/automl.ipynb

```diff
@@ -454,6 +454,7 @@
     "automl_with_ensembling = AutoMLSearch(X_train=X, y_train=y,\n",
     "                                      problem_type=\"binary\",\n",
     "                                      allowed_model_families=[ModelFamily.RANDOM_FOREST, ModelFamily.LINEAR_MODEL],\n",
+    "                                      train_best_pipeline=False,\n",
```

Comment:
To speed up the build?

Reply:
Ah, was left over from when we got an error with label encoder. Removing!